metrics/ddl.go

```diff
@@ -123,6 +123,22 @@ var (
 			Name:      "backfill_percentage_progress",
 			Help:      "Percentage progress of backfill",
 		}, []string{LblType})
+
+	DDLJobTableDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "tidb",
+		Subsystem: "ddl",
+		Name:      "job_table_duration_seconds",
+		Help:      "Bucketed histogram of processing time (s) of the 3 DDL job tables",
+		Buckets:   prometheus.ExponentialBuckets(0.001, 2, 20), // 1ms ~ 524s
+	}, []string{LblType})
+
+	DDLRunningJobCount = prometheus.NewGaugeVec(
```

Comment:
Could we use `JobsGauge` instead of this? We can add a `running` label for it.

Reply:
running job itself also has label, split will make more clear I think